domain/entity/latest_entry.go

```diff
@@ -0,0 +1,90 @@
+package entity
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/appengine/datastore"
+)
+
+const (
+	LatestEntryCodeTamai    = "tamai-sd"
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const LatestEntryCodeTamai should have comment (or a comment on this block) or be unexported